### PR TITLE
Updates to User Access docs

### LINKED ISSUE #520: Add Access Overview page to docs
We had an existing Access Overview page that wasn't published.

Review Access Overview page and any access related docs page, and update section to make platform permissions easier to understand.

### PR DESCRIPTION
**Pull request by heitortsergent**
Preview: https://meta.stoplight.io/docs/platform/branches/heitor-permissions-update/638203faacd8c-access-overview

Making some updates to the User Access section:

- Adding an Access Overview page (was already created by @pamgoodrich, I just made some edits to it)
- Removed Allowed Email Domains page (we have a section in Manage Workspace Members with the same information)
- Reorganized ToC:
	- Removed Workspace Access group and moved articles up a level
	- Moved API & Article Visibility topic into User Access group

Related to #484. Closes #520.

Blocked by deployment of https://github.com/stoplightio/platform-internal/pull/13124 so we don't break any links in the platform.

### CODE REVIEW

File: 2.-workspaces/p.access-overview.md [hunk 1]
```diff
@@ -1,29 +1,125 @@
...
+
+There are several components that Stoplight provides to control user access at different levels:
+
+- Workspace Access
+- Project Access
+- Group Access
+
+You can use different access levels to have fine-grained control on which users can access your Stoplight projects. You can also assign users different roles, to control the permissions a user has in the workspace and projects.
+
+In addition to access level, there are also visibility controls that can affect what a user might be able to access in the platform:
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:15]: Maybe change "access" here to "see"?

File: 2.-workspaces/p.access-overview.md [hunk 2]
```diff
@@ -1,29 +1,125 @@
...
+
+- Group Visibility
+- Project Visibility
+- Article and API Visibility
+
+Using visibility controls allows users to control which projects are visible to everyone (such as a public documentation portal), internally to all workspace users and guests, or private to a limited subset of users in your workspace. You can also control the visibility settings for articles and APIs, which can be especially useful when developing a public API with private or work-in-progress endpoints, for example.
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:21]: "for articles and APIs within any project"?

File: 2.-workspaces/p.access-overview.md [hunk 3]
```diff
@@ -1,29 +1,125 @@
...
+
+The different access levels and visibility options exist for different use cases. For example, if you have an API description in Stoplight and want to collaborate with other team members internally, workspace and project access might be all you need.
+
+If you have an API description and you would like to have public documentation, but you also have certain API endpoints that you want to keep hidden while they're being developed, then you can use the Article and API Visibility controls to achieve that.
+
+---
+
+Let's take a look at all the access and visibility options.
 
 ## Workspace Access
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:35]: Maybe create a section with ## levels for Access and Visibility, then organize under that.

## Access Guidelines
### Workspace Access
### Group Access
### Project Access

## Visibility Guidelines
### Project Visibility
### Group Visibility
### API and Article Visibility

File: 2.-workspaces/p.access-overview.md (hunk 2 above)
Comment (heitortsergent) [reply 2.-workspaces/p.access-overview.md:21]: If you mark something as internal it should also hide it from workspace guests.

File: 2.-workspaces/o.group-access.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Manage Group Access
 
-Use a combination of visibility type (public and internal) and roles to assign access to sets of projects organized in groups.
+Use a combination of visibility type (public and internal) and roles to assign access to sets of projects organized in [groups](m.groups.md).
```
Comment (pamgoodrich) [inline 2.-workspaces/o.group-access.md:3]: Maybe change "visibility type" to "project visibility type"?

File: 2.-workspaces/p.access-overview.md [hunk 5]
```diff
@@ -1,29 +1,121 @@
...
+Let's take a look at all the access and visibility options.
 
-Use a combination of [visibility type](o.group-access.md#group-visibility-and-access) (public and internal) and [group roles](o.group-access.md#group-roles) to assign bulk access to sets of projects.
+## Access Guidelines
+
+### Workspace Access
+
+Workspace-wide access is provided through roles:
+
+- **Guest** - for external users to your organization that need limited access on a project-by-project basis.
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:33]: I think the "for" after the hyphen should be "For". What do you think?

File: 2.-workspaces/p.access-overview.md [hunk 6]
```diff
@@ -1,29 +1,121 @@
...
+
+You can set the role when you [invite members](d.inviting-your-team.md) to your workspace.
+
+### Project Access
+
+While workspace roles define workspace-wide access, project access allows you to have granular control over who can manage, edit, or view a project.
+
+User can have the following roles in a project:
+
+- **Guest** - can view the project and files that aren't marked as internal.
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:52]: Same as above. I think "can" should be "Can"

File: 2.-workspaces/p.access-overview.md [hunk 7]
```diff
@@ -1,29 +1,121 @@
...
+
+For example, if you have a public API and would like to provide documentation for all your users, you can set your project visibility to **Public** and it'll be accessible by anyone with the workspace URL.
+
+For more details about project visibility, head to the [Manage Project Access](l.project-roles.md) page.
+
+### Group Visibility
+
+Similar to [project visibility](l.project-roles.md), group visibility defines what account types can see or join a group. It also impacts what kinds of projects can belong to that group. There are two options for group visibility:
+
+- **Public**: the group and any public projects in them are visible to everyone, including logged-out users. Use public groups to organize your public-facing API documentation.
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:102]: Same as above. "the" should be "The"

File: 2.-workspaces/p.access-overview.md [hunk 8]
```diff
@@ -1,29 +1,121 @@
...
+
+Articles, APIs, and parts of an API description (paths and models) can be marked as internal to allow users to hide them from external and internal users with guest access.
+
+This is useful for users that have public documentation, but also have internal documents, or APIs and endpoints, that are still being developed, or are only used internally, but live side-by-side with other public documentation or APIs. This allows users to manage public projects while still keeping any sensitive information private.
+
+For more details about article and API visibility, head to the [Article and API visibility](../4.-documentation/set-internal-docs.md) page.
+
+## Git Provider Permissions and Service Accounts
+
+For users that integrate with Git providers, a user's permission in their Git provider can also have impact when accessing a project in Stoplight. For example, if a user doesn't have edit permissions in a Git repository, they will also not be able to edit a Stoplight project that's connected to that repository.
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:117]: "users that" > "users who"

Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:117]: "if a user doesn't have edit permissions in a Git repository, they will also not be able to edit a Stoplight project that's connected to that repository."

"Users who do not have edit permissions in a Git repository cannot edit a Stoplight project connected to that repository."

File: 2.-workspaces/p.access-overview.md [hunk 9]
```diff
@@ -1,29 +1,121 @@
...
+
+The exception to that rule is if you're using service accounts. By connecting a service account to a Stoplight project, you can allow users that don't have a Git account, or don't have edit permissions in a repository, to use Stoplight to edit and make contributions to that project.
```
Comment (pamgoodrich) [inline 2.-workspaces/p.access-overview.md:119]: "users that" > "users who"

**Review by pamgoodrich — APPROVED** [review]

File: 2.-workspaces/o.group-access.md (hunk 4 above)
Comment (heitortsergent) [reply 2.-workspaces/o.group-access.md:3]: This is supposed to be group visibility type. I can add "group" there instead to make it clearer?

**Pull request merged**